### PR TITLE
Cleanup formatting

### PR DESCRIPTION
**Pull request by adamschachne**

### CODE REVIEW

File: src/compiler/diagnosticMessages.json [hunk 1]
```diff
@@ -1007,7 +1007,6 @@
         "category": "Error",
         "code": 1349
     },
-
```
Comment (jcs090218) [inline src/compiler/diagnosticMessages.json]: I think these lines are just better for navigating the file? Like move between empty lines so you can navigate the file faster.

File: src/compiler/checker.ts [hunk 2]
```diff
@@ -10902,7 +10904,7 @@ namespace ts {
             return reportedError;
         }
 
-        function *generateJsxAttributes(node: JsxAttributes): ElaborationIterator {
```
Comment (jcs090218) [inline src/compiler/checker.ts]: Like these are base on MS's own coding standard. I do not exactly know what MS's coding standard is but if this match might be a great fix?

File: src/compiler/checker.ts [hunk 3]
```diff
@@ -3170,11 +3170,13 @@ namespace ts {
                     enclosingDeclaration,
                     flags: flags || NodeBuilderFlags.None,
                     // If no full tracker is provided, fake up a dummy one with a basic limited-functionality moduleResolverHost
-                    tracker: tracker && tracker.trackSymbol ? tracker : { trackSymbol: noop, moduleResolverHost: flags! & NodeBuilderFlags.DoNotIncludeSymbolChain ? {
```
Comment (jcs090218) [inline src/compiler/checker.ts]: I think these code are just base on their coding standard so no need to change at all?

Comment (jcs090218) [inline src/compiler/checker.ts]: I think these are just base on their coding standard so no need to change at all?

File: src/compiler/checker.ts (hunk 2 above)
Comment (jcs090218) [inline src/compiler/checker.ts]: Having asterisk before/after is base MS's coding standard. I do not exactly know what MS's coding standard is but if this match might be a good fix?

File: src/compiler/diagnosticMessages.json (hunk 1 above)
Comment (jcs090218) [inline src/compiler/diagnosticMessages.json]: I think having these empty lines are just better to navigate the file. Like moving cursor between the empty line. Not surely know these lines has to be removed.

File: src/compiler/checker.ts (hunk 2 above)
Comment (adamschachne) [reply src/compiler/checker.ts]: I did a global search for function * and function* to see what the preferred style was. I am seeing both however..

Comment (jcs090218) [reply src/compiler/checker.ts]: Well, if that is the case, either we will have to wait for someone to review this or we can just completely ignore this. Anyway both options would not harm the result. :D

**Comment by RyanCavanaugh** [comment]
This seems to be different without being better.

**Pull request closed without merging**